recommendation-app/flink-deployment.yaml

```diff
@@ -3,7 +3,7 @@ kind: FlinkDeployment
 metadata:
   name: recommendation-app
 spec:
-  image:  flink-sql-runner:latest
+  image: quay.io/streamshub/flink-sql-runner:latest
   imagePullPolicy: Never
```

Comment:
We should remove the imagePullPolicy as well

Reply:
good point!